sdk/iot/Azure.Iot.Hub.Service/src/Generated/Models/DeviceIdentity.cs

```diff
@@ -21,20 +21,20 @@ public DeviceIdentity()
         }
 
         /// <summary> Initializes a new instance of DeviceIdentity. </summary>
-        /// <param name="deviceId"> . </param>
-        /// <param name="generationId"> . </param>
-        /// <param name="etag"> . </param>
-        /// <param name="connectionState"> . </param>
-        /// <param name="status"> . </param>
-        /// <param name="statusReason"> . </param>
-        /// <param name="connectionStateUpdatedTime"> . </param>
-        /// <param name="statusUpdatedTime"> . </param>
-        /// <param name="lastActivityTime"> . </param>
-        /// <param name="cloudToDeviceMessageCount"> . </param>
-        /// <param name="authentication"> . </param>
-        /// <param name="capabilities"> Status of Capabilities enabled on the device. </param>
-        /// <param name="deviceScope"> . </param>
-        /// <param name="parentScopes"> . </param>
+        /// <param name="deviceId"> The unique identifier of the device. </param>
+        /// <param name="generationId"> The IoT Hub-generated, case-sensitive string up to 128 characters long. This value is used to distinguish devices with the same deviceId, when they have been deleted and re-created. </param>
+        /// <param name="etag"> The string representing a weak ETag for the device identity, as per RFC7232. </param>
+        /// <param name="connectionState"> The state of the device. </param>
+        /// <param name="status"> The status of the device. If the status disabled, a device cannot connect to the service. </param>
+        /// <param name="statusReason"> The 128 character-long string that stores the reason for the device identity status. All UTF-8 characters are allowed. </param>
+        /// <param name="connectionStateUpdatedTime"> The date and time the connection state was last updated. </param>
+        /// <param name="statusUpdatedTime"> The date and time when the status field was last updated. </param>
+        /// <param name="lastActivityTime"> The date and last time the device last connected, received, or sent a message. </param>
+        /// <param name="cloudToDeviceMessageCount"> The number of cloud-to-device messages currently queued to be sent to the device. </param>
+        /// <param name="authentication"> The authentication mechanism used by the device. </param>
+        /// <param name="capabilities"> The set of capabilities of the device. For example, if this device is an edge device or not. </param>
+        /// <param name="deviceScope"> The scope of the device. Auto generated and immutable for edge devices and modifiable in leaf devices to create child/parent relationship. </param>
+        /// <param name="parentScopes"> The scopes of the upper level edge devices if applicable. Only available for edge devices. </param>
```

Comment:
FYI - this is apparently under a FF as well, correct? So shouldn't be in this swagger doc.
They might have already made a note of this, but worth tracking it from our side as well.